docs/resources/bgp_instance.md

```diff
@@ -26,24 +26,24 @@ resource "mikrotik_bgp_instance" "instance" {
 
 ### Optional
 
-- `client_to_client_reflection` (Boolean) The comment of the IP Pool to be created. Default: `true`.
+- `client_to_client_reflection` (Boolean) In case this instance is a route reflector: whether to redistribute routes learned from one routing reflection client to other clients.
 - `cluster_id` (String) In case this instance is a route reflector: cluster ID of the router reflector cluster this instance belongs to.
 - `comment` (String) The comment of the BGP instance to be created.
 - `confederation` (Number) In case of BGP confederations: autonomous system number that identifies the [local] confederation as a whole.
 - `confederation_peers` (String) List of AS numbers internal to the [local] confederation. For example: `10,20,30-50`.
 - `disabled` (Boolean) Whether instance is disabled.
-- `ignore_as_path_len` (Boolean) Whether to ignore AS_PATH attribute in BGP route selection algorithm. Default: `false`.
-- `out_filter` (String) Output routing filter chain used by all BGP peers belonging to this instance. Default: `""`.
-- `redistribute_connected` (Boolean) If enabled, this BGP instance will redistribute the information about connected routes. Default: `false`.
-- `redistribute_ospf` (Boolean) If enabled, this BGP instance will redistribute the information about routes learned by OSPF. Default: `false`.
-- `redistribute_other_bgp` (Boolean) If enabled, this BGP instance will redistribute the information about routes learned by other BGP instances. Default: `false`.
-- `redistribute_rip` (Boolean) If enabled, this BGP instance will redistribute the information about routes learned by RIP. Default: `false`.
-- `redistribute_static` (Boolean) If enabled, the router will redistribute the information about static routes added to its routing database. Default: `false`.
-- `routing_table` (String) Name of routing table this BGP instance operates on.  Default: `""`.
+- `ignore_as_path_len` (Boolean) Whether to ignore AS_PATH attribute in BGP route selection algorithm.
+- `out_filter` (String) Output routing filter chain used by all BGP peers belonging to this instance.
+- `redistribute_connected` (Boolean) If enabled, this BGP instance will redistribute the information about connected routes.
+- `redistribute_ospf` (Boolean) If enabled, this BGP instance will redistribute the information about routes learned by OSPF.
+- `redistribute_other_bgp` (Boolean) If enabled, this BGP instance will redistribute the information about routes learned by other BGP instances.
+- `redistribute_rip` (Boolean) If enabled, this BGP instance will redistribute the information about routes learned by RIP.
+- `redistribute_static` (Boolean) If enabled, the router will redistribute the information about static routes added to its routing database.
+- `routing_table` (String) Name of routing table this BGP instance operates on.
```

Comment:
Can we update the `Description` string to include the default value? I don't know why tfplugindocs changed that, but I thought it was nice to have those in the docs.

Reply:
It is [not supported](https://github.com/hashicorp/terraform-plugin-docs/issues/65) by `tfplugindocs` yet.
Related feature was requested in [terraform-plugin-framework](https://github.com/hashicorp/terraform-plugin-framework/issues/625) to have at least workaround for this.
I will take a look - maybe it is possible to contribute to solve this issue 🤷

Reply:
@ddelnano I just found a way to add `Default` to the docs.
We need to implement custom type for each Terraform attribute type (`bool, string, int64, set, list`) like this:
```go
type stringDefault struct {
	schema.StringAttribute
}

func (a stringDefault) GetDescription() string {
	origDescription := a.StringAttribute.GetDescription()
	if a.Default == nil {
		return origDescription
	}

	resp := defaults.StringResponse{}
	a.Default.DefaultString(context.TODO(), defaults.StringRequest{}, &resp)
	origDescription = fmt.Sprintf(`%s Default: %s`, origDescription, resp.PlanValue)

	return origDescription
}
```

and use it as a wrapper in the `Schema()` method (example is based on `mikrotik_bgp_peer` resource):
```go
			"address_families": stringDefault{
				schema.StringAttribute{
					Optional: true,
					Computed: true,
					PlanModifiers: []planmodifier.String{
						stringplanmodifier.UseStateForUnknown(),
					},
					Default:     stringdefault.StaticString("ip"),
					Description: "The list of address families about which this peer will exchange routing information.",
				},
			},
```

Don't look at namings :)

Reply:
@maksym-nazarenko nice, that sounds like a great approach until upstream exposes equivalent functionality for the plugin framwork 👍

Reply:
@ddelnano the docs default value issue will be handled in #203 and #204, so this PR can be reviewed on its own.